packages/help-center/src/components/utils.tsx

```diff
@@ -51,8 +51,13 @@ export const getLastMessage = ( {
 };
 
 export const getZendeskConversations = () => {
-	const conversations = Smooch?.getConversations?.() ?? [];
-	return conversations as unknown as ZendeskConversation[];
+	try {
+		const conversations = Smooch?.getConversations?.() ?? [];
+		return conversations as unknown as ZendeskConversation[];
+	} catch {
+		// Smooch is not completely initialized yet
+		return [];
+	}
```

Comment:
These try catch is not really needed but it helps if someone have our same issue